fs/inode/fs_files.c

```diff
@@ -800,6 +800,10 @@ int fdlist_copy(FAR struct fdlist *plist, FAR struct fdlist *clist,
                   file_put(filep);
                   continue;
                 }
+
```

Comment:
@Donny9 could you please include a comment here? I think fdcheck_restore() will called if spawn_file_is_duplicateable() succeed, right?

BTW, it is a pleasure when the Kconfig help enlighten us:

```
config FDCHECK
        bool "Enable fdcheck"
        default n
        depends on LIBC_OPEN_MAX <= 256
        ---help---
                Enable the fdcheck support
```

😂

Reply:
BTW, let's move this change to new patch.

Reply:
> @Donny9 could you please include a comment here? I think fdcheck_restore() will be called if spawn_file_is_duplicateable() succeed, right?
> 
> BTW, it is a pleasure when the Kconfig help enlighten us:
> 
> ```
> config FDCHECK
>         bool "Enable fdcheck"
>         default n
>         depends on LIBC_OPEN_MAX <= 256
>         ---help---
>                 Enable the fdcheck support
> ```
> 
> 😂
fdcheck_restore is used to restore fd before calling fdlist_install line 823.

Reply:
> BTW, let's move this change to new patch.

Done~